packages/web/src/scss/components/Stack/README.md

```diff
@@ -1,5 +1,9 @@
 # Stack
 
+[Stack] is a component that allows you to compose elements vertically.
```

Comment:
what is the benefit of this link? :D

Reply:
Copypasta from `web-twig` module. Thanks. ;)